### PR TITLE
#11 add weather title component

### LINKED ISSUE #11: Add weather title component

### PR DESCRIPTION
**Pull request by mrpeterhanley**
closes #11 

The component is used like this:

`// <WeatherTitle value="Shower" />`

weather-title-component

### CODE REVIEW

**Comment by inovux** [comment]
Looks good to me, can you add a screenshot of your final component as well :)? I think that would be a nice last thing to add to the PR's

**Comment by inovux** [comment]
Looks good to me, just update to the latest `develop` branch

**Comment by mrpeterhanley** [comment]
Updated with latest changes from Develop branch and added screenshot

File: src/components/WeatherTitle/WeatherTitle.module.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.weatherTitle {
+  font-weight: 600;
+  font-size: 36px;
+  line-height: 42.26px;
+  text-align: center;
+  color: #a09fb1;
```
Comment (hananina) [inline src/components/WeatherTitle/WeatherTitle.module.css]: I think you can replace #a09fb1 with css variable after merging latest develop into this branch🙋‍♀️

File: src/App.tsx [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
-function App() {
+export const App: FC = () => {
   return (
     <div>
       <h1>Weather App</h1>
     </div>
   )
 }
-
-export default App
```
Comment (hananina) [inline src/App.tsx]: @mrpeterhanley Can we remove this line😃? I'm ok If it's working fine🙏

File: public/index.html [hunk 3]
```diff
@@ -24,7 +25,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Weather App</title>
```
Comment (hananina) [inline public/index.html]: this PR seems includes changes not related to whether title component, after merging the latest develop branch😀? Is this a problem caused by the merge?

**Comment by hananina** [comment]
@mrpeterhanley I saw this changes and it looks good! just left comments about what I notice. Please check it🙏🙏

File: src/components/index.tsx [hunk 4]
```diff
@@ -0,0 +1 @@
+export { WeatherTitle } from './WeatherTitle/index'
```
Comment (inovux) [inline src/components/index.tsx]: You dont need the /index here by the way

**Review by inovux — CHANGES_REQUESTED** [review]

**Comment by inovux** [comment]
Did you copy paste the changes from develop into your own branch? Since it seems like it shows that you made all the changes yourself somehow. If so can you actually try to rebase your branch from develop?

**Comment by mrpeterhanley** [comment]
@inovux I used git checkout develop filename to add / update the new files from develop. I haven't learnt rebase yet but I will learn how to use it....

**Review by inovux — APPROVED** [review]

**Review by hananina — APPROVED** [review]
Looks good to me🙌🙌

**Pull request merged**